docs/core-manage-asdf-vm.md

```diff
@@ -33,6 +33,14 @@ To use the latest changes, you can point Homebrew to the master branch of the re
 brew install asdf --HEAD
 ```
 
+### --Pacman--
+
+Install using `pacman`:
+
+```shell
+pacman -S asdf-vm
+```
+
```

Comment:
Hmm, `asdf-vm` is in AUR, you need to either [build and install it manually](https://wiki.archlinux.org/index.php/Arch_User_Repository#Installing_packages) or [use an AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers) like [`yay`](https://github.com/Jguer/yay/).